.infrastructure/security/rbac.yml

```diff
@@ -0,0 +1,34 @@
+kind: ServiceAccount
+apiVersion: v1
+metadata:
+  name: secrets-lister
+  namespace: todoapp
+
+---
+
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  namespace: todoapp
+  name: secret-lister
+rules:
+- apiGroups: [""]
+  resources: ["secrets"]
+  verbs: ["list", "get"]
+
+---
+
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: secret-lister-binding
+  namespace: todoapp
+subjects:
+- kind: ServiceAccount
+  name: secrets-lister
+  namespace: todoapp
```

Comment:
You don't need to duplicate namespace here. You put it in metadata

Reply:
removed extra namespace in [remove 'namespace' in RoleBinding, subjects](https://github.com/mate-academy/devops_todolist_kubernetes_task_12_rbac/pull/23/commits/6987b76fb5cd724bf7ea375c3531a39f661f3805) commit